app/assets/javascripts/create.js

```diff
@@ -0,0 +1,32 @@
+$(() => {
+    createIdea()
+})
+
+var createIdea = () => {
```

Comment:
I might consider naming this something else. What does this function actually do? It _does not_ create an idea, right? Rather, it add event listeners to the idea creation form, right? So, I might consider naming it something like `bindEventsToNewIdeaForm`. Sure, it's verbose, but it's also super clear what this function does.